utils/server-side-props-helper/error-handler.ts

```diff
@@ -43,6 +49,12 @@ const handleExceptions = (exception: any, req: IncomingMessage | undefined) => {
       exception instanceof NotASiretError
     ) {
       return redirectPageNotFound(message, scope);
+    } else if (exception instanceof IsLikelyAnIdRnfException) {
+      return redirectIfIdRnf(message);
+    } else if (exception instanceof NotAnIdRnfError) {
+      return redirectIdRnfInvalid(message);
+    } else if (exception instanceof RnfNotFoundError) {
+      return redirectIdRnfIntrouvable(message);
     } else if (exception instanceof HttpNotFound) {
       return redirectPageNotFound(message, scope);
     } else if (exception instanceof SearchEngineError) {
```

Comment:
It's a bit weird for me to use exception in that case. I'm not sure it should be handled as such, but instead, a well crafted type return type and a switch case in the top of the route. But this is outside of the scope of this PR